listenbrainz/listenstore/influx_listenstore.py

```diff
@@ -534,6 +534,7 @@ def dump_listens(self, location, dump_time=datetime.today(), threads=DUMP_DEFAUL
 
             pxz.stdin.close()
 
+        pxz.wait()
```

Comment:
I think the absence of this might have been the cause of the dump file corruption. We didn't wait for the pxz command to exit, leading to a race condition between the cp and this, leading to corrupted files in some places.

I came across this because the hashes created and printed were different from the hashes of the actual files in dev also.